code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -270,7 +270,7 @@
 						return W.afterattack(target,src)
 
 			var/randn = rand(1, 100)
-			randn = max(1, randn - H.stats.getStat(STAT_ROB))
+			randn = max(1, randn - H.stats.getStat(STAT_ROB) + src.stats.getStat(STAT_TGH))//Occulus Edit: TGH makes you harder to shove
```

Comment:
BYOND evaluates addition before subtraction (ref 10.5 https://secure.byond.com/docs/guide/chap06.html) -- in a theoretical fight of 50 ROB vs 50 TGH and a randn var of 33, 50 will be added to 50 for 100, and then 33 - 100 = -67, which will result in the `max` selecting 1 and guaranteeing a knockdown.

Please encapsulate the subtraction to make it evaluate first -- so in that same theoretical `randn` 33 w/ ROB 50 vs TGH 50, it becomes (33 - 50) + 50 for a final value of 33.

Reply:
Figure 6.10 summarizes the order of operations of the mathematical symbols from highest to lowest. Operators on the same line have equal precedence and are therefore evaluated as they appear in the expression from left to right. 


( ) ++ -- -(negation)
--
**
* / %
+ -